### DISCUSSION TITLE
UX imrovements for DocPage Organizer: Drug & drop and Adaptive thumbnails grid

### DISCUSSION THREAD

**Discussion by hotenov** [JakubMelka/PDF4QT#91] [Ideas]
1) Add drug & drop for DocPage Organizer in order to add images or other PDF files into workspace.

In current PDF4QT version `v1.3.5` we have to add images from file choosing dialog window. It's not convenient, considering that recent location (folder) is reset to default after the app has been restarted (_I use portable version on Windows_)

2) _(It would be nice)_ Make thumbnails grid more adaptive to window size.

Horizontal scrolling is not so convenient as vertical. And we won't have to Zoom In / Zoom Out each time when we want to arrange our thumbnails to new window size _(see GIF below to understand what I'm talking about)_.

![2023-09-13 Thumbnails grid is not adaptive to windows size](https://github.com/JakubMelka/PDF4QT/assets/11818507/dbf741dd-756b-48bb-ad2b-6b1ae1f2b12e)

**Comment by JakubMelka** [comment]
Hello @hotenov, I have written issue #97, I hope I will have time next week to implement it.
